crates/ironrdp-graphics/src/lib.rs

```diff
@@ -17,3 +17,19 @@ pub mod subband_reconstruction;
 pub mod zgfx;
 
 mod utils;
+
+pub fn rfx_encode_component(
+    input: &mut [i16],
+    output: &mut [u8],
+    quant: &ironrdp_pdu::codecs::rfx::Quant,
+    mode: ironrdp_pdu::codecs::rfx::EntropyAlgorithm,
+) -> Result<usize, rlgr::RlgrError> {
+    assert_eq!(input.len(), 64 * 64);
+
+    let mut temp = [0; 64 * 64]; // size = 8k, too big?
+
+    dwt::encode(input, temp.as_mut_slice());
+    quantization::encode(input, quant);
+    subband_reconstruction::encode(&mut input[4032..]);
+    rlgr::encode(mode, input, output)
+}
```

Comment:
Sure, we could move it there. What's the rule though? because the whole related code could be moved then :) Imho, the RFX encoding algorithm isn't *complete* without this function, so it makes sense to have it as part of the graphics crate. I don't mind much at this point, so if you prefer to move it, that's ok.

Reply:
I see… to be honest I think we should split this crate at some point, until then it’s probably best as you suggested